tests/plugins/autocomplete/manual/completingkeycodes.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: 4.10.0, feature, tp3560
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, basicstyles, autocomplete, textmatch
```

Comment:
After rebasing the branch, the `textmatch` plugin will be loaded out of the box by `__template__.html`.